DSESwift/DataStorageEngine.swift

```diff
@@ -100,12 +98,14 @@ public class DataStorageEngine{
     * Initialize DSE and start the timer for periodic syncing.
     **/
     public func start() throws {
-        if (self.config.sessionId == nil || self.config.appKey == nil || self.config.userId == nil) {
+        if (self.config.sessionId == "" || self.config.appKey == "" || self.config.userId == "") {
```

Comment:
@kanekin swift have isEmpty function to check if a string is empty

https://developer.apple.com/library/ios/documentation/Swift/Conceptual/Swift_Programming_Language/StringsAndCharacters.html#//apple_ref/doc/uid/TP40014097-CH7-XID_368